tests/test_router_bgp.rb

```diff
@@ -639,19 +637,16 @@ def log_neighbor_changes(bgp)
     bgp.destroy
   end
 
-  def test_log_neighbor_changes_not_configured
-    asnum = 55
-    bgp = RouterBgp.new(asnum)
-    refute(bgp.log_neighbor_changes,
-           'bgp log_neighbor_changes should be disabled')
-    bgp.destroy
-  end
-
   def test_default_log_neighbor_changes
-    asnum = 55
-    bgp = RouterBgp.new(asnum)
-    refute(bgp.default_log_neighbor_changes,
-           'bgp log_neighbor_changes default value should be false')
+    bgp = setup_default
+    if platform == :ios_xr
```

Comment:
Right. That's why I want you to check the default value in order to know whether to assert or refute.